examples/react-simple/src/ix/Transaction.js

```diff
@@ -1,37 +1,162 @@
 import React, {useState} from "react"
 import * as sdk from "@onflow/sdk"
+import * as t from "@onflow/types"
 import {signingFunction} from "./utils/signing-function.js"
+import {authorizationFunction} from "./utils/authorization-function.js"
 
 export const Transaction = () => {
   const [result, setResult] = useState(null)
 
   const run = async () => {
 
-    const acctResponse = await sdk.send(await sdk.pipe(await sdk.build([
-      sdk.getAccount("f8d6e0586b0a20c7")
+    /*
+
+      Transaction
+      -----------
+
+      Transactions are declared by building an interaction and specifying a transaction within it.
+      This is done by specifying Cadence code within the transaction builder.
+
+      Transactions also require other pieces of information as so they can be sent.
+
+      Proposer, Payer and Authorizers
+      --------------------------------
+
+      Transactions require a proposer, payer and an array of authorizers.
+
+      - The proposer specifies the Account for which the sequence number of one of it's keys will be used and incremented by this transaction.
+      - The payer specifies the Account for which will pay for the transaction.
+      - Each authorizer specifies an AuthAccount which can be used within the cadence code of the transaction.
```

Comment:
```suggestion
      - Each authorizer specifies an AuthAccount which can be used within the Cadence code of the transaction.
```

Reply:
I wonder if this needs to be more clear, or show an example. I know what this means, but It seems kind of out of place here.